src/client/entry.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
 import Festival from "./components/Festival";
+import "!style!css!sass!./scss/style.scss";
```

Comment:
This is where I tell webpack to import the sass contents and include it in the page that will be generated.